test-fixtures/mssql/sample_table.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE [dbo].sample_table (
+  id INT NOT NULL,
+  PRIMARY KEY (id)
+);
```

Comment:
I am not so sure this samble table is necessary for your "underscorde_no_fields" table.
You can pick any other existing table for you foreign key, it will avoid to add to much code.

Reply:
Yeah that's what I did at first, but adding a new references to these tables for my specific test breaks all others, as test checks are based on raw diffs with an expected file. I felt like it was better than fixing a working test.

Tell me.